components/kyma-environment-broker/internal/process/provisioning/create_runtime_without_kyma_step.go

```diff
@@ -59,28 +61,43 @@ func (s *CreateRuntimeWithoutKymaStep) Run(operation internal.Operation, log log
 		return s.operationManager.OperationFailed(operation, "invalid operation data - cannot create provisioning input", err, log)
 	}
 
-	log.Infof("call ProvisionRuntime: kubernetesVersion=%s, region=%s, provider=%s, name=%s",
-		requestInput.ClusterConfig.GardenerConfig.KubernetesVersion,
-		requestInput.ClusterConfig.GardenerConfig.Region,
-		requestInput.ClusterConfig.GardenerConfig.Provider,
-		requestInput.ClusterConfig.GardenerConfig.Name)
+	if requestInput.ClusterConfig.GardenerConfig != nil {
+		log.Infof("call ProvisionRuntime: kubernetesVersion=%s, region=%s, provider=%s, name=%s",
+			requestInput.ClusterConfig.GardenerConfig.KubernetesVersion,
+			requestInput.ClusterConfig.GardenerConfig.Region,
+			requestInput.ClusterConfig.GardenerConfig.Provider,
+			requestInput.ClusterConfig.GardenerConfig.Name)
+	}
 
-	provisionerResponse, err := s.provisionerClient.ProvisionRuntime(operation.ProvisioningParameters.ErsContext.GlobalAccountID, operation.ProvisioningParameters.ErsContext.SubAccountID, requestInput)
-	switch {
-	case kebError.IsTemporaryError(err):
-		log.Errorf("call to provisioner failed (temporary error): %s", err)
-		return operation, 5 * time.Second, nil
-	case err != nil:
-		log.Errorf("call to Provisioner failed: %s", err)
-		return s.operationManager.OperationFailed(operation, "call to the provisioner service failed", err, log)
+	var runtimeId string
+	var provisionerOperationId string
+	if operation.ProvisioningParameters.PlanID == broker.OwnClusterPlanID {
```

Comment:
I don't like the fact that step "create runtime without kyma" is not creating runtime for special plan. I'd prefer to not run such step in such condition